pulp_docker/app/serializers.py

```diff
@@ -162,3 +162,20 @@ class Meta:
         model = models.DockerDistribution
         fields = tuple(set(RepositoryVersionDistributionSerializer.Meta.fields) - {'base_url'}) + (
             'registry_path',)
+
+
+class DockerRecursiveAddSerializer(serializers.Serializer):
+    """
+    Serializer for adding content to a Docker repository.
+    """
+
+    repository = serializers.HyperlinkedRelatedField(
```

Comment:
let's also add repo_version?

Reply:
On second thought, what is the difference in behavior if we pass repository version? Oneshot upload for instance does not accept repo version.

Reply:
i think if a repository is passed then latest version is used, otherwise new repo_version will be built on top of a specifically provided one

Reply:
oneshot upload does not accept, but for example rpm_copy does. i think it is up to us to decide whether we want both options or we stick to just repo

Reply:
repo version makes more sense for copy than for add. Adding will always just create a new repo version. For copy, it might make sense to `copy from` a version, but `copy to` seems to only need a repository and not a version.

Reply:
we agreed to go with repository only